### DISCUSSION TITLE
Alpine Expression Error: Cannot read properties of undefined (reading 'add')

### DISCUSSION THREAD

**Discussion by otatechie** [Power-Components/livewire-powergrid#1088] [Q&A]
I was able to implement the table but anytime I select the checkbox, I get this error message. What am I doing wrong?

<img width="1349" alt="Screenshot 2023-08-08 at 6 41 29 PM" src="https://github.com/Power-Components/livewire-powergrid/assets/30342985/c155ea9c-8192-45ab-a8eb-15ae966f1eb4">

**Comment by Kings-Israel** [comment]
Found a solution yet?
